site/en/_partials/privacy-sandbox/glossary-entries/scaling.md

```diff
@@ -0,0 +1 @@
+[Scaling](/docs/privacy-sandbox/private-aggregation-fundamentals/#noise-and-scaling) refers to multiplying your aggregatable value by a number you choose to adjust the impact of noise. It impacts the effect of noise and the use of your contribution budget.
```

Comment:
We may want to drop "Scaling". For the most part, we use it in reference to aggregation, but it could also be used in other enlarging of infrastructure.

Reply:
yes was not sure about this

Reply:
resolving this comment and re-asking so Sam can find easily